diff --git a/homeassistant-supervised/DEBIAN/postinst b/homeassistant-supervised/DEBIAN/postinst
index 19e6c60..e175910 100755
--- a/homeassistant-supervised/DEBIAN/postinst
+++ b/homeassistant-supervised/DEBIAN/postinst
@@ -19,14 +19,6 @@ URL_VERSION="https://version.home-assistant.io/stable.json"
 HASSIO_VERSION=$(curl -s ${URL_VERSION} | jq -e -r '.supervisor')
 URL_APPARMOR_PROFILE="https://version.home-assistant.io/apparmor.txt"
 
-# reload systemd
-info "Reload systemd"
-systemctl daemon-reload
-
-# Restart NetworkManager
-info "Restarting NetworkManager"
-systemctl restart "${SERVICE_NM}"
-
 # Set permissions of /etc/systemd/resolved.conf
 # check if file has correct permissions
 if [ "$(stat -c %a /etc/systemd/resolved.conf)" != "644" ]; then
@@ -37,8 +29,6 @@ fi
 # Enable and restart systemd-resolved
 info "Enable systemd-resolved"
 systemctl enable systemd-resolved.service> /dev/null 2>&1;
-info "Restarting systemd-resolved"
-systemctl restart systemd-resolved.service
 
 # Check and fix systemd-journal-gatewayd socket location
 if [ ! -S "/run/systemd-journal-gatewayd.sock" ]; then
@@ -60,10 +50,6 @@ if [ "$(systemctl is-active nfs-utils.service)" = 'inactive' ]; then
     systemctl start nfs-utils.service
 fi
 
-# Restart Docker service
-info "Restarting docker service"
-systemctl restart "${SERVICE_DOCKER}"
-
 # Check network connection
 while ! curl -q ${URL_CHECK_ONLINE} >/dev/null 2>&1 ; do
     info "Waiting for ${URL_CHECK_ONLINE} - network interface might be down..."
@@ -153,11 +139,6 @@ sed -i -e "s,%%SERVICE_DOCKER%%,${SERVICE_DOCKER},g" \
 
 chmod a+x "${PREFIX}/sbin/hassio-apparmor"
 systemctl enable hassio-apparmor.service > /dev/null 2>&1;
-systemctl start hassio-apparmor.service
-
-# Start Supervisor
-info "Start Home Assistant Supervised"
-systemctl start hassio-supervisor.service
 
 # Install HA CLI
 info "Installing the 'ha' cli"
diff --git a/homeassistant-supervised/DEBIAN/preinst b/homeassistant-supervised/DEBIAN/preinst
index c36cf74..b0881c2 100755
--- a/homeassistant-supervised/DEBIAN/preinst
+++ b/homeassistant-supervised/DEBIAN/preinst
@@ -47,7 +47,6 @@ fi
 # Check dmesg access
 if [[ "$(sysctl --values kernel.dmesg_restrict)" != "0" ]]; then
     info "Fix kernel dmesg restriction"
-    echo 0 > /proc/sys/kernel/dmesg_restrict
     echo "kernel.dmesg_restrict=0" >> /etc/sysctl.conf
 fi
 
diff --git a/homeassistant-supervised/etc/NetworkManager/NetworkManager.conf b/homeassistant-supervised/etc/NetworkManager/NetworkManager.conf
index 343203c..ffcaeb1 100644
--- a/homeassistant-supervised/etc/NetworkManager/NetworkManager.conf
+++ b/homeassistant-supervised/etc/NetworkManager/NetworkManager.conf
@@ -5,7 +5,7 @@ autoconnect-retries-default=0
 rc-manager=file
 
 [keyfile]
-unmanaged-devices=type:bridge;type:tun;driver:veth
+unmanaged-devices=type:bridge;type:tun;driver:veth;interface-name:enx00e04c68012d;type=macvlan
 
 [logging]
 backend=journal
@@ -19,4 +19,4 @@ uri=http://checkonline.home-assistant.io/online.txt
 interval=600
 
 [device]
-wifi.scan-rand-mac-address=no
\ No newline at end of file
+wifi.scan-rand-mac-address=no
